app/forms/steps/client/benefit_check_result_form.rb

```diff
@@ -2,17 +2,31 @@ module Steps
   module Client
     class BenefitCheckResultForm < Steps::BaseFormObject
       include Steps::HasOneAssociation
+      attribute :passporting_benefit, :value_object, source: YesNoAnswer
```

Comment:
Wouldn't this be the other way around? the yes-no attribute is the one used in the radio options shown in the view.
The other attribute is something internal is it? Maybe I'm missing something here.

Reply:
You're not missing anything. My initial instinct was to declare the passporting_benefit as a boolean, but I noticed the ```YesNoAnswer``` was being used elsewhere and thought I should use that, especially as the benefit check result is either a 'yes' or 'no'

Reply:
Perhaps if this is to store the result of the benefit checker, a boolean might be better? This is assuming the result would be an absolute pass/fail. If we think we could have different nuanced results, like a success, a fail, and a "maybe" (just an example) then a string is best so we can store the result returned from the benefit checker in our DB and be able to propagate that to the case worker if they need it, or to perform different logic?

Reply:
I'll change it to a boolean for now, and if it turns to be more nuanced, we can change it later.